src/components/common/Chart/ChartBody.tsx

```diff
@@ -28,8 +29,15 @@ export default function ChartBody(props: Props) {
   )
   const height = props.height ?? 'h-100'
 
+  const renderTooltipContent = (payload: ChartDataPayloadProps[]) => {
+    const value = Number(payload[0].value) ?? 0
+    return (
+      <Text size='sm'>{formatValue(value, { minDecimals: 0, maxDecimals: 0, prefix: '$' })}</Text>
+    )
+  }
+
```

Comment:
```suggestion
  function TooltipContent(props: {payload: ChartDataPayloadProps[]}) {
    const { payload } = props
    const value = Number(payload[0].value) ?? 0
    return (
      <Text size='sm'>{formatValue(value, { minDecimals: 0, maxDecimals: 0, prefix: '$' })}</Text>
    ) 
```

Convert this function into a component function.